dll/cpl/desk/lang/bg-BG.rc

```diff
@@ -33,7 +33,7 @@ STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
 CAPTION "Екранен предпазител"
 FONT 8, "MS Shell Dlg", 0, 0
 BEGIN
-    CONTROL "", IDC_SCREENS_PREVIEW, "Static", SS_OWNERDRAW, 70, 10, 105, 70, WS_EX_STATICEDGE
```

Comment:
Is this needed then, the trailing "zero" for the style? (does it still work without it?)

Reply:
I just copypasted an existing control. But yes, it seems to compile fine without it.

Reply:
Fixed the styles in master and here.